lib/helper/WebDriverIO.js

```diff
@@ -1216,4 +1220,16 @@ function withStrictLocator(locator) {
   }
 }
 
+function hashCode(str) {
```

Comment:
pleease move that to `lib/utils` with some comments about how the hashing is done and for what purpose. Because it would be hard to understand that for later users